pilot/pkg/xds/eds.go

```diff
@@ -377,11 +387,11 @@ func (s *DiscoveryServer) pushEds(push *model.PushContext, con *Connection, vers
 	edsPushes.Increment()
 
 	if edsUpdatedServices == nil {
-		adsLog.Infof("EDS: PUSH for node:%s clusters:%d endpoints:%d empty:%v",
-			con.proxy.ID, len(con.Clusters()), endpoints, empty)
+		adsLog.Infof("EDS: PUSH for node:%s clusters:%d endpoints:%d empty:%v cached:%v/%v",
```

Comment:
I guess you just change the log level for debug

Reply:
I did not change the log level - maybe you were looking at old commit?